@shared/constants/mercury.ts

```diff
@@ -0,0 +1,3 @@
+export const INDEXER_URLS: { [index: string]: string } = {
+  TESTNET: "http://localhost:3002/api/v1",
```

Comment:
We don't have env vars yet and this isn't really a secret but maybe this should go in an env var?